topi/tests/python/test_topi_vision.py

```diff
@@ -46,7 +96,7 @@ def check_device(device):
         f(tvm_data, tvm_valid_count, tvm_out)
         tvm.testing.assert_allclose(tvm_out.asnumpy(), np_result, rtol=1e-4)
 
-    for device in ['llvm', 'opencl', 'cuda']:
```

Comment:
Does this one fail with 'opencl' and 'cuda'?

Reply:
get_valid_counts need to be implemented for gpu targets.